### PR TITLE
feat(about): add new page for third party licenses

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
## Description

* Add html file with all the licenses involved in fossology.

## How to test

* Install FOSSology and check third party license license page.

### CODE REVIEW

**Comment by mcjaeger** [comment]
ehem? 
Screenshot 2020-11-19 at 09 26 33

**Comment by mcjaeger** [comment]
updated the info also in about menu item. tested with vagrant. looks OK on my side.

File: src/www/ui/page/AboutPage.php [hunk 1]
```diff
@@ -57,8 +57,7 @@ protected function handle(Request $request)
     $vars = array(
         'licenseCount' => $this->licenseDao->getLicenseCount(),
         'project' => _("FOSSology"),
-        'copyright' => _("Copyright (C) 2007-2014 Hewlett-Packard Development Company, L.P.<br>\nCopyright (C) 2014-2017 Siemens AG."),
-    );
+  );
```
Comment (GMishx) [inline src/www/ui/page/AboutPage.php:60]: Can you please fix the indentation by adding 2 more spaces?

**Review by GMishx — CHANGES_REQUESTED** [review]
Also, the docker tests are failing. I investigated a little bit and found there is a `NOTICES*` entry in `.dockerignore`. Can you please change that entry to `NOTICES` and `NOTICES.spdx`?

**Review by GMishx — APPROVED** [review]

**Pull request merged**